### PR TITLE
Vanilla Vehicles Expanded Patches

### PR DESCRIPTION
**Pull request by Tostov**
## Additions

Added patches for Vanilla Vehicles Expanded, for races and bodydef

## Reasoning

Melee is roughly centipede level for the "strongest" vehicles, scaling down for other ones. They are cars after all, and although it's probably a little over scaled, I don't think this is either a serious mod, or a serious concern. Also cars should be fairly dangerous if they run into a human, even if not at full speed.

## Testing

Check tests you have performed:
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded

### CODE REVIEW

**Review by MMMiracles — DISMISSED** [review]
We can argue the delicate balance of man vs car when it comes to the authenticity CE strides to add but after testing I can confirm that it has no issues in the error logging. I can also confirm that you can technically win a fist fight against a car if you manage to punch out the tires fast enough.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]

File: Patches/Vanilla Animals Expanded - Vehicles/ThingDefs_Races/Races_AFVehicles.xml [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+				  <xpath>/Defs/ThingDef[
+				  defName="VAEAF_CarFamily" or
+				  defName="VAEAF_CarDelivery" or
+				  defName="VAEAF_CarVintage" or
+				  defName="VAEAF_CarPickup" or
+				  defName="VAEAF_CarForklift"
+				  ]</xpath>
+				  <value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+					  <bodyShape>QuadrupedLow</bodyShape>
```
[the remark this answers is not in the document]
Comment (sumghai) [reply Patches/Vanilla Animals Expanded - Vehicles/ThingDefs_Races/Races_AFVehicles.xml:21]: Yes, it is currently defined as follows in BodyShapes.xml:

```
<CombatExtended.BodyShapeDef>
	<defName>Vehicle</defName>
	<width>1</width>
	<widthLaying>1</widthLaying>
	<height>1</height>
	<heightLaying>1</heightLaying>
</CombatExtended.BodyShapeDef>
```

File: Patches/Vanilla Animals Expanded - Vehicles/ThingDefs_Races/Races_AFVehicles.xml [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+					<ArmorRating_Blunt>6</ArmorRating_Blunt>
+				  </value>
+				</li>
+				
+				<li Class="PatchOperationReplace">
+				  <xpath>/Defs/ThingDef[defName="VAEAF_CarPickup"]/tools</xpath>
+				  <value>
+				    <tools>
+					  <li Class="CombatExtended.ToolCE">
+						<label>head</label>
```
Comment (N7Huntsman) [inline Patches/Vanilla Animals Expanded - Vehicles/ThingDefs_Races/Races_AFVehicles.xml:143]: This should probably be swapped to something else, given that the vehicles don't have "heads". Something like grille? body?

**Review by N7Huntsman — CHANGES_REQUESTED** [review]

File: Patches/Vanilla Animals Expanded - Vehicles/ThingDefs_Races/Races_AFVehicles.xml (hunk 2 above)
Comment (Tostov) [reply Patches/Vanilla Animals Expanded - Vehicles/ThingDefs_Races/Races_AFVehicles.xml]: Ahh, nice catch. Should be fixed now.

**Review by N7Huntsman — APPROVED** [review]

**Comment by N7Huntsman** [comment]
@MMMiracles If you think it's good to go, please put an Approval on it under reviews, and I'll get it merged.

**Review by ghost — APPROVED** [review]

**Pull request merged**